[INFO ][20161201 17:54:11,692][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1f89ab83: startup date [Thu Dec 01 17:54:11 CST 2016]; root of context hierarchy
[INFO ][20161201 17:54:11,744][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 17:54:11,981][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 17:54:12,051][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 17:54:12,096][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 17:54:12,384][PropertiesLoaderSupport:172] Loading properties file from class path resource [jdbc.properties]
[WARN ][20161201 17:54:12,386][AbstractApplicationContext:546] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanInitializationException: Could not load properties; nested exception is java.io.FileNotFoundException: class path resource [jdbc.properties] cannot be opened because it does not exist
[ERROR][20161201 17:54:12,390][DubboProvider:17] == DubboProvider context start error:
org.springframework.beans.factory.BeanInitializationException: Could not load properties; nested exception is java.io.FileNotFoundException: class path resource [jdbc.properties] cannot be opened because it does not exist
	at org.springframework.context.support.PropertySourcesPlaceholderConfigurer.postProcessBeanFactory(PropertySourcesPlaceholderConfigurer.java:148)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:284)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:166)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:678)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:520)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at dubbo.test.DubboProvider.main(DubboProvider.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: java.io.FileNotFoundException: class path resource [jdbc.properties] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
	at org.springframework.core.io.support.EncodedResource.getInputStream(EncodedResource.java:153)
	at org.springframework.core.io.support.PropertiesLoaderUtils.fillProperties(PropertiesLoaderUtils.java:98)
	at org.springframework.core.io.support.PropertiesLoaderSupport.loadProperties(PropertiesLoaderSupport.java:175)
	at org.springframework.core.io.support.PropertiesLoaderSupport.mergeProperties(PropertiesLoaderSupport.java:156)
	at org.springframework.context.support.PropertySourcesPlaceholderConfigurer.postProcessBeanFactory(PropertySourcesPlaceholderConfigurer.java:139)
	... 12 more
[INFO ][20161201 17:57:20,458][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1dfe2924: startup date [Thu Dec 01 17:57:20 CST 2016]; root of context hierarchy
[INFO ][20161201 17:57:20,549][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 17:57:21,069][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 17:57:21,108][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 17:57:21,155][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 17:57:21,417][PropertiesLoaderSupport:172] Loading properties file from class path resource [db.properties]
[INFO ][20161201 17:57:21,418][PropertiesLoaderSupport:172] Loading properties file from class path resource [public_system.properties]
[WARN ][20161201 17:57:21,745][AbstractApplicationContext:546] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userFacade': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBiz': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.fengyu.service.user.dao.UserDao] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
[ERROR][20161201 17:57:21,759][DubboProvider:17] == DubboProvider context start error:
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userFacade': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBiz': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.fengyu.service.user.dao.UserDao] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:311)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1214)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:543)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:772)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:839)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:538)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at dubbo.test.DubboProvider.main(DubboProvider.java:13)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBiz': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.fengyu.service.user.dao.UserDao] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:311)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1214)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:543)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.autowireResource(CommonAnnotationBeanPostProcessor.java:512)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.getResource(CommonAnnotationBeanPostProcessor.java:486)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor$ResourceElement.getResourceToInject(CommonAnnotationBeanPostProcessor.java:615)
	at org.springframework.beans.factory.annotation.InjectionMetadata$InjectedElement.inject(InjectionMetadata.java:169)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:308)
	... 13 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.fengyu.service.user.dao.UserDao] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@javax.annotation.Resource(shareable=true, lookup=, name=, description=, authenticationType=CONTAINER, type=class java.lang.Object, mappedName=)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1373)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1119)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1014)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.autowireResource(CommonAnnotationBeanPostProcessor.java:508)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.getResource(CommonAnnotationBeanPostProcessor.java:486)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor$ResourceElement.getResourceToInject(CommonAnnotationBeanPostProcessor.java:615)
	at org.springframework.beans.factory.annotation.InjectionMetadata$InjectedElement.inject(InjectionMetadata.java:169)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:308)
	... 26 more
[INFO ][20161201 18:01:13,063][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1f89ab83: startup date [Thu Dec 01 18:01:13 CST 2016]; root of context hierarchy
[INFO ][20161201 18:01:13,115][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 18:01:13,521][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 18:01:13,567][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 18:01:13,610][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 18:01:13,896][PropertiesLoaderSupport:172] Loading properties file from class path resource [db.properties]
[INFO ][20161201 18:01:13,897][PropertiesLoaderSupport:172] Loading properties file from class path resource [public_system.properties]
[INFO ][20161201 18:01:14,233][Slf4jMLog$Slf4jMLogger$InfoLogger:212] MLog clients using slf4j logging.
[INFO ][20161201 18:01:15,332][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
[WARN ][20161201 18:01:15,692][AbstractApplicationContext:546] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userFacade': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBiz': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userDao': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.alibaba.druid.pool.DruidDataSource com.fengyu.common.core.dao.BaseDaoImpl.druidDataSource; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.alibaba.druid.pool.DruidDataSource] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
[WARN ][20161201 18:01:15,700][DisposableBeanAdapter:364] Invocation of destroy method 'close' failed on bean with name 'sessionTemplate': java.lang.UnsupportedOperationException: Manual close is not allowed over a Spring managed SqlSession
[ERROR][20161201 18:01:15,702][DubboProvider:17] == DubboProvider context start error:
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userFacade': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBiz': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userDao': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.alibaba.druid.pool.DruidDataSource com.fengyu.common.core.dao.BaseDaoImpl.druidDataSource; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.alibaba.druid.pool.DruidDataSource] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:311)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1214)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:543)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:772)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:839)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:538)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at dubbo.test.DubboProvider.main(DubboProvider.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userBiz': Injection of resource dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userDao': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.alibaba.druid.pool.DruidDataSource com.fengyu.common.core.dao.BaseDaoImpl.druidDataSource; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.alibaba.druid.pool.DruidDataSource] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:311)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1214)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:543)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.autowireResource(CommonAnnotationBeanPostProcessor.java:512)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.getResource(CommonAnnotationBeanPostProcessor.java:486)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor$ResourceElement.getResourceToInject(CommonAnnotationBeanPostProcessor.java:615)
	at org.springframework.beans.factory.annotation.InjectionMetadata$InjectedElement.inject(InjectionMetadata.java:169)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:308)
	... 18 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'userDao': Injection of autowired dependencies failed; nested exception is org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.alibaba.druid.pool.DruidDataSource com.fengyu.common.core.dao.BaseDaoImpl.druidDataSource; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.alibaba.druid.pool.DruidDataSource] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:334)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1214)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:543)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.autowireResource(CommonAnnotationBeanPostProcessor.java:512)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.getResource(CommonAnnotationBeanPostProcessor.java:486)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor$ResourceElement.getResourceToInject(CommonAnnotationBeanPostProcessor.java:615)
	at org.springframework.beans.factory.annotation.InjectionMetadata$InjectedElement.inject(InjectionMetadata.java:169)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.context.annotation.CommonAnnotationBeanPostProcessor.postProcessPropertyValues(CommonAnnotationBeanPostProcessor.java:308)
	... 31 more
Caused by: org.springframework.beans.factory.BeanCreationException: Could not autowire field: private com.alibaba.druid.pool.DruidDataSource com.fengyu.common.core.dao.BaseDaoImpl.druidDataSource; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.alibaba.druid.pool.DruidDataSource] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:573)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:331)
	... 44 more
Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [com.alibaba.druid.pool.DruidDataSource] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1373)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1119)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1014)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:545)
	... 46 more
[INFO ][20161201 18:20:43,806][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1dfe2924: startup date [Thu Dec 01 18:20:43 CST 2016]; root of context hierarchy
[INFO ][20161201 18:20:43,897][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 18:20:44,464][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 18:20:44,498][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 18:20:44,540][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 18:20:44,781][PropertiesLoaderSupport:172] Loading properties file from class path resource [db.properties]
[INFO ][20161201 18:20:44,782][PropertiesLoaderSupport:172] Loading properties file from class path resource [public_system.properties]
[INFO ][20161201 18:20:45,239][Slf4jMLog$Slf4jMLogger$InfoLogger:212] MLog clients using slf4j logging.
[INFO ][20161201 18:20:46,360][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
[INFO ][20161201 18:20:47,096][ServiceBean:107]  [DUBBO] The service ready on spring started. service: com.fengyu.facade.user.service.UserFacade, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:47,236][ServiceConfig:510]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to local registry, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:47,237][ServiceConfig:470]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:47,238][ServiceConfig:481]  [DUBBO] Register dubbo service com.fengyu.facade.user.service.UserFacade url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111 to registry registry://139.196.225.215:2181/com.alibaba.dubbo.registry.RegistryService?application=fengyu-service-user&dubbo=2.5.3&pid=15684&registry=zookeeper&timestamp=1480587647104, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:47,700][AbstractServer:69]  [DUBBO] Start NettyServer bind /0.0.0.0:20881, export /192.168.171.1:20881, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:47,738][ZkEventThread:64] Starting ZkClient event thread.
[INFO ][20161201 18:20:47,746][Environment:100] Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
[INFO ][20161201 18:20:47,747][Environment:100] Client environment:host.name=windows10.microdone.cn
[INFO ][20161201 18:20:47,748][Environment:100] Client environment:java.version=1.8.0_101
[INFO ][20161201 18:20:47,748][Environment:100] Client environment:java.vendor=Oracle Corporation
[INFO ][20161201 18:20:47,748][Environment:100] Client environment:java.home=D:\Program Files\Java\jdk1.8.0_101\jre
[INFO ][20161201 18:20:47,749][Environment:100] Client environment:java.class.path=D:\Program Files\Java\jdk1.8.0_101\jre\lib\charsets.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\deploy.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\access-bridge-64.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\cldrdata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\dnsns.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jaccess.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jfxrt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\localedata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\nashorn.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunec.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunjce_provider.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunmscapi.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunpkcs11.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\zipfs.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\javaws.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jce.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfr.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfxswt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jsse.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\management-agent.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\plugin.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\resources.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\rt.jar;D:\workspace\fengyu\fengyu-service-user\target\test-classes;D:\workspace\fengyu\fengyu-service-user\target\classes;D:\workspace\fengyu\fengyu-common-core\target\classes;D:\workspace\fengyu\fengyu-common\target\classes;D:\mavenLocal\com\jcraft\jsch\0.1.50\jsch-0.1.50.jar;D:\workspace\fengyu\fengyu-common-config\target\classes;D:\workspace\fengyu\fengyu-facade-user\target\classes;D:\mavenLocal\junit\junit\4.12\junit-4.12.jar;D:\mavenLocal\org\hamcrest\hamcrest-core\1.3\hamcrest-core-1.3.jar;D:\mavenLocal\log4j\log4j\1.2.17\log4j-1.2.17.jar;D:\mavenLocal\net\sf\json-lib\json-lib\2.0\json-lib-2.0-jdk15.jar;D:\mavenLocal\commons-beanutils\commons-beanutils\1.7.0\commons-beanutils-1.7.0.jar;D:\mavenLocal\commons-collections\commons-collections\3.2\commons-collections-3.2.jar;D:\mavenLocal\commons-logging\commons-logging\1.1\commons-logging-1.1.jar;D:\mavenLocal\net\sf\ezmorph\ezmorph\1.0.2\ezmorph-1.0.2.jar;D:\mavenLocal\org\json\json\20151123\json-20151123.jar;D:\mavenLocal\commons-lang\commons-lang\2.6\commons-lang-2.6.jar;D:\mavenLocal\org\springframework\spring-context\4.2.4.RELEASE\spring-context-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aop\4.2.4.RELEASE\spring-aop-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-beans\4.2.4.RELEASE\spring-beans-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-core\4.2.4.RELEASE\spring-core-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-expression\4.2.4.RELEASE\spring-expression-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-context-support\4.2.4.RELEASE\spring-context-support-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aspects\4.2.4.RELEASE\spring-aspects-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-tx\4.2.4.RELEASE\spring-tx-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-jdbc\4.2.4.RELEASE\spring-jdbc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-orm\4.2.4.RELEASE\spring-orm-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-web\4.2.4.RELEASE\spring-web-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-webmvc\4.2.4.RELEASE\spring-webmvc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-test\4.2.4.RELEASE\spring-test-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-remoting\2.0.8\spring-remoting-2.0.8.jar;D:\mavenLocal\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;D:\mavenLocal\org\aspectj\aspectjweaver\1.7.3\aspectjweaver-1.7.3.jar;D:\mavenLocal\cglib\cglib\2.2.2\cglib-2.2.2.jar;D:\mavenLocal\asm\asm\3.3.1\asm-3.3.1.jar;D:\mavenLocal\org\freemarker\freemarker\2.3.23\freemarker-2.3.23.jar;D:\mavenLocal\com\alibaba\druid\1.0.14\druid-1.0.14.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\jconsole.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\tools.jar;D:\mavenLocal\com\mchange\c3p0\0.9.5.2\c3p0-0.9.5.2.jar;D:\mavenLocal\com\mchange\mchange-commons-java\0.2.11\mchange-commons-java-0.2.11.jar;D:\mavenLocal\mysql\mysql-connector-java\5.1.38\mysql-connector-java-5.1.38.jar;D:\mavenLocal\redis\clients\jedis\2.4.2\jedis-2.4.2.jar;D:\mavenLocal\org\apache\commons\commons-pool2\2.0\commons-pool2-2.0.jar;D:\mavenLocal\net\sf\ehcache\ehcache\2.10.1\ehcache-2.10.1.jar;D:\mavenLocal\org\mybatis\mybatis\3.2.8\mybatis-3.2.8.jar;D:\mavenLocal\org\mybatis\mybatis-spring\1.2.2\mybatis-spring-1.2.2.jar;D:\mavenLocal\javax\persistence\persistence-api\1.0\persistence-api-1.0.jar;D:\mavenLocal\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar;D:\mavenLocal\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar;D:\mavenLocal\commons-codec\commons-codec\1.6\commons-codec-1.6.jar;D:\mavenLocal\commons-net\commons-net\3.2\commons-net-3.2.jar;D:\mavenLocal\org\jdom\jdom\1.1.3\jdom-1.1.3.jar;D:\mavenLocal\com\github\pagehelper\pagehelper\4.1.5\pagehelper-4.1.5.jar;D:\mavenLocal\com\github\jsqlparser\jsqlparser\0.9.5\jsqlparser-0.9.5.jar;D:\mavenLocal\javax\servlet\javax.servlet-api\3.1.0\javax.servlet-api-3.1.0.jar;D:\mavenLocal\commons-fileupload\commons-fileupload\1.2.1\commons-fileupload-1.2.1.jar;D:\mavenLocal\commons-io\commons-io\1.3.2\commons-io-1.3.2.jar;D:\mavenLocal\com\alibaba\dubbo\2.5.3\dubbo-2.5.3.jar;D:\mavenLocal\org\javassist\javassist\3.15.0-GA\javassist-3.15.0-GA.jar;D:\mavenLocal\org\jboss\netty\netty\3.2.5.Final\netty-3.2.5.Final.jar;D:\mavenLocal\org\apache\zookeeper\zookeeper\3.4.6\zookeeper-3.4.6.jar;D:\mavenLocal\jline\jline\0.9.94\jline-0.9.94.jar;D:\mavenLocal\io\netty\netty\3.7.0.Final\netty-3.7.0.Final.jar;D:\mavenLocal\com\101tec\zkclient\0.7\zkclient-0.7.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-core\2.4.3\jackson-core-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-databind\2.4.3\jackson-databind-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-annotations\2.4.0\jackson-annotations-2.4.0.jar;D:\mavenLocal\net\sourceforge\jexcelapi\jxl\2.6.12\jxl-2.6.12.jar;D:\mavenLocal\org\jsoup\jsoup\1.7.3\jsoup-1.7.3.jar;D:\mavenLocal\org\apache\tomcat\servlet-api\6.0.37\servlet-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\jsp-api\6.0.37\jsp-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\el-api\6.0.37\el-api-6.0.37.jar;D:\mavenLocal\org\slf4j\slf4j-api\1.7.5\slf4j-api-1.7.5.jar;D:\mavenLocal\org\slf4j\slf4j-log4j12\1.7.5\slf4j-log4j12-1.7.5.jar;D:\Program Files\JetBrains\IntelliJ IDEA 2016.1\lib\idea_rt.jar
[INFO ][20161201 18:20:47,750][Environment:100] Client environment:java.library.path=D:\Program Files\Java\jdk1.8.0_101\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\NetSarang;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x64;C:\Program Files\Lenovo\Fingerprint Manager Pro\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;D:\Program Files\TortoiseSVN\bin;D:\Program Files\Java\jdk1.8.0_101\bin;D:\Program Files\Java\jdk1.8.0_101\jre\bin;D:\maven-3.0.5/bin;D:\maven-3.0.5\bin;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;.
[INFO ][20161201 18:20:47,750][Environment:100] Client environment:java.io.tmpdir=C:\Users\admin\AppData\Local\Temp\
[INFO ][20161201 18:20:47,750][Environment:100] Client environment:java.compiler=<NA>
[INFO ][20161201 18:20:47,751][Environment:100] Client environment:os.name=Windows 7
[INFO ][20161201 18:20:47,751][Environment:100] Client environment:os.arch=amd64
[INFO ][20161201 18:20:47,751][Environment:100] Client environment:os.version=6.1
[INFO ][20161201 18:20:47,751][Environment:100] Client environment:user.name=admin
[INFO ][20161201 18:20:47,751][Environment:100] Client environment:user.home=C:\Users\admin
[INFO ][20161201 18:20:47,752][Environment:100] Client environment:user.dir=D:\workspace\fengyu
[INFO ][20161201 18:20:47,753][ZooKeeper:438] Initiating client connection, connectString=139.196.225.215:2181 sessionTimeout=30000 watcher=org.I0Itec.zkclient.ZkClient@55f8669d
[INFO ][20161201 18:20:47,802][ZkClient:934] Waiting for keeper state SyncConnected
[INFO ][20161201 18:20:47,807][ClientCnxn$SendThread:975] Opening socket connection to server 139.196.225.215/139.196.225.215:2181. Will not attempt to authenticate using SASL (unknown error)
[INFO ][20161201 18:20:47,843][ClientCnxn$SendThread:852] Socket connection established to 139.196.225.215/139.196.225.215:2181, initiating session
[INFO ][20161201 18:20:47,877][ClientCnxn$SendThread:1235] Session establishment complete on server 139.196.225.215/139.196.225.215:2181, sessionid = 0x158b9767d180002, negotiated timeout = 30000
[INFO ][20161201 18:20:47,879][ZkClient:711] zookeeper state changed (SyncConnected)
[INFO ][20161201 18:20:47,881][AbstractRegistry:302]  [DUBBO] Register: dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:47,997][AbstractRegistry:325]  [DUBBO] Subscribe: provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:20:48,137][AbstractRegistry:422]  [DUBBO] Notify urls for subscribe url provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111, urls: [empty://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111], dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:30:57,286][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1dfe2924: startup date [Thu Dec 01 18:30:57 CST 2016]; root of context hierarchy
[INFO ][20161201 18:30:57,368][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 18:30:57,859][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 18:30:57,888][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 18:30:57,925][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 18:30:58,292][PropertiesLoaderSupport:172] Loading properties file from class path resource [db.properties]
[INFO ][20161201 18:30:58,293][PropertiesLoaderSupport:172] Loading properties file from class path resource [public_system.properties]
[INFO ][20161201 18:30:58,626][Slf4jMLog$Slf4jMLogger$InfoLogger:212] MLog clients using slf4j logging.
[INFO ][20161201 18:30:59,399][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
[INFO ][20161201 18:31:00,248][ServiceBean:107]  [DUBBO] The service ready on spring started. service: com.fengyu.facade.user.service.UserFacade, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:00,367][ServiceConfig:510]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to local registry, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:00,367][ServiceConfig:470]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:00,368][ServiceConfig:481]  [DUBBO] Register dubbo service com.fengyu.facade.user.service.UserFacade url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262 to registry registry://139.196.225.215:2181/com.alibaba.dubbo.registry.RegistryService?application=fengyu-service-user&dubbo=2.5.3&pid=17032&registry=zookeeper&timestamp=1480588260254, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:00,861][AbstractServer:69]  [DUBBO] Start NettyServer bind /0.0.0.0:20881, export /192.168.171.1:20881, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:00,883][AbstractRegistry:232]  [DUBBO] Load registry store file C:\Users\admin\.dubbo\dubbo-registry-139.196.225.215.cache, data: {com.fengyu.facade.user.service.UserFacade=empty://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15684&side=provider&timestamp=1480587647111}, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:00,902][ZkEventThread:64] Starting ZkClient event thread.
[INFO ][20161201 18:31:00,908][Environment:100] Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
[INFO ][20161201 18:31:00,910][Environment:100] Client environment:host.name=windows10.microdone.cn
[INFO ][20161201 18:31:00,910][Environment:100] Client environment:java.version=1.8.0_101
[INFO ][20161201 18:31:00,910][Environment:100] Client environment:java.vendor=Oracle Corporation
[INFO ][20161201 18:31:00,910][Environment:100] Client environment:java.home=D:\Program Files\Java\jdk1.8.0_101\jre
[INFO ][20161201 18:31:00,910][Environment:100] Client environment:java.class.path=D:\Program Files\Java\jdk1.8.0_101\jre\lib\charsets.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\deploy.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\access-bridge-64.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\cldrdata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\dnsns.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jaccess.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jfxrt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\localedata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\nashorn.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunec.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunjce_provider.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunmscapi.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunpkcs11.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\zipfs.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\javaws.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jce.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfr.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfxswt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jsse.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\management-agent.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\plugin.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\resources.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\rt.jar;D:\workspace\fengyu\fengyu-service-user\target\test-classes;D:\workspace\fengyu\fengyu-service-user\target\classes;D:\workspace\fengyu\fengyu-common-core\target\classes;D:\workspace\fengyu\fengyu-common\target\classes;D:\mavenLocal\com\jcraft\jsch\0.1.50\jsch-0.1.50.jar;D:\workspace\fengyu\fengyu-common-config\target\classes;D:\workspace\fengyu\fengyu-facade-user\target\classes;D:\mavenLocal\junit\junit\4.12\junit-4.12.jar;D:\mavenLocal\org\hamcrest\hamcrest-core\1.3\hamcrest-core-1.3.jar;D:\mavenLocal\log4j\log4j\1.2.17\log4j-1.2.17.jar;D:\mavenLocal\net\sf\json-lib\json-lib\2.0\json-lib-2.0-jdk15.jar;D:\mavenLocal\commons-beanutils\commons-beanutils\1.7.0\commons-beanutils-1.7.0.jar;D:\mavenLocal\commons-collections\commons-collections\3.2\commons-collections-3.2.jar;D:\mavenLocal\commons-logging\commons-logging\1.1\commons-logging-1.1.jar;D:\mavenLocal\net\sf\ezmorph\ezmorph\1.0.2\ezmorph-1.0.2.jar;D:\mavenLocal\org\json\json\20151123\json-20151123.jar;D:\mavenLocal\commons-lang\commons-lang\2.6\commons-lang-2.6.jar;D:\mavenLocal\org\springframework\spring-context\4.2.4.RELEASE\spring-context-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aop\4.2.4.RELEASE\spring-aop-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-beans\4.2.4.RELEASE\spring-beans-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-core\4.2.4.RELEASE\spring-core-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-expression\4.2.4.RELEASE\spring-expression-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-context-support\4.2.4.RELEASE\spring-context-support-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aspects\4.2.4.RELEASE\spring-aspects-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-tx\4.2.4.RELEASE\spring-tx-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-jdbc\4.2.4.RELEASE\spring-jdbc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-orm\4.2.4.RELEASE\spring-orm-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-web\4.2.4.RELEASE\spring-web-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-webmvc\4.2.4.RELEASE\spring-webmvc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-test\4.2.4.RELEASE\spring-test-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-remoting\2.0.8\spring-remoting-2.0.8.jar;D:\mavenLocal\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;D:\mavenLocal\org\aspectj\aspectjweaver\1.7.3\aspectjweaver-1.7.3.jar;D:\mavenLocal\cglib\cglib\2.2.2\cglib-2.2.2.jar;D:\mavenLocal\asm\asm\3.3.1\asm-3.3.1.jar;D:\mavenLocal\org\freemarker\freemarker\2.3.23\freemarker-2.3.23.jar;D:\mavenLocal\com\alibaba\druid\1.0.14\druid-1.0.14.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\jconsole.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\tools.jar;D:\mavenLocal\com\mchange\c3p0\0.9.5.2\c3p0-0.9.5.2.jar;D:\mavenLocal\com\mchange\mchange-commons-java\0.2.11\mchange-commons-java-0.2.11.jar;D:\mavenLocal\mysql\mysql-connector-java\5.1.38\mysql-connector-java-5.1.38.jar;D:\mavenLocal\redis\clients\jedis\2.4.2\jedis-2.4.2.jar;D:\mavenLocal\org\apache\commons\commons-pool2\2.0\commons-pool2-2.0.jar;D:\mavenLocal\net\sf\ehcache\ehcache\2.10.1\ehcache-2.10.1.jar;D:\mavenLocal\org\mybatis\mybatis\3.2.8\mybatis-3.2.8.jar;D:\mavenLocal\org\mybatis\mybatis-spring\1.2.2\mybatis-spring-1.2.2.jar;D:\mavenLocal\javax\persistence\persistence-api\1.0\persistence-api-1.0.jar;D:\mavenLocal\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar;D:\mavenLocal\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar;D:\mavenLocal\commons-codec\commons-codec\1.6\commons-codec-1.6.jar;D:\mavenLocal\commons-net\commons-net\3.2\commons-net-3.2.jar;D:\mavenLocal\org\jdom\jdom\1.1.3\jdom-1.1.3.jar;D:\mavenLocal\com\github\pagehelper\pagehelper\4.1.5\pagehelper-4.1.5.jar;D:\mavenLocal\com\github\jsqlparser\jsqlparser\0.9.5\jsqlparser-0.9.5.jar;D:\mavenLocal\javax\servlet\javax.servlet-api\3.1.0\javax.servlet-api-3.1.0.jar;D:\mavenLocal\commons-fileupload\commons-fileupload\1.2.1\commons-fileupload-1.2.1.jar;D:\mavenLocal\commons-io\commons-io\1.3.2\commons-io-1.3.2.jar;D:\mavenLocal\com\alibaba\dubbo\2.5.3\dubbo-2.5.3.jar;D:\mavenLocal\org\javassist\javassist\3.15.0-GA\javassist-3.15.0-GA.jar;D:\mavenLocal\org\jboss\netty\netty\3.2.5.Final\netty-3.2.5.Final.jar;D:\mavenLocal\org\apache\zookeeper\zookeeper\3.4.6\zookeeper-3.4.6.jar;D:\mavenLocal\jline\jline\0.9.94\jline-0.9.94.jar;D:\mavenLocal\io\netty\netty\3.7.0.Final\netty-3.7.0.Final.jar;D:\mavenLocal\com\101tec\zkclient\0.7\zkclient-0.7.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-core\2.4.3\jackson-core-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-databind\2.4.3\jackson-databind-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-annotations\2.4.0\jackson-annotations-2.4.0.jar;D:\mavenLocal\net\sourceforge\jexcelapi\jxl\2.6.12\jxl-2.6.12.jar;D:\mavenLocal\org\jsoup\jsoup\1.7.3\jsoup-1.7.3.jar;D:\mavenLocal\org\apache\tomcat\servlet-api\6.0.37\servlet-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\jsp-api\6.0.37\jsp-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\el-api\6.0.37\el-api-6.0.37.jar;D:\mavenLocal\org\slf4j\slf4j-api\1.7.5\slf4j-api-1.7.5.jar;D:\mavenLocal\org\slf4j\slf4j-log4j12\1.7.5\slf4j-log4j12-1.7.5.jar;D:\Program Files\JetBrains\IntelliJ IDEA 2016.1\lib\idea_rt.jar
[INFO ][20161201 18:31:00,912][Environment:100] Client environment:java.library.path=D:\Program Files\Java\jdk1.8.0_101\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\NetSarang;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x64;C:\Program Files\Lenovo\Fingerprint Manager Pro\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;D:\Program Files\TortoiseSVN\bin;D:\Program Files\Java\jdk1.8.0_101\bin;D:\Program Files\Java\jdk1.8.0_101\jre\bin;D:\maven-3.0.5/bin;D:\maven-3.0.5\bin;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;.
[INFO ][20161201 18:31:00,912][Environment:100] Client environment:java.io.tmpdir=C:\Users\admin\AppData\Local\Temp\
[INFO ][20161201 18:31:00,912][Environment:100] Client environment:java.compiler=<NA>
[INFO ][20161201 18:31:00,913][Environment:100] Client environment:os.name=Windows 7
[INFO ][20161201 18:31:00,913][Environment:100] Client environment:os.arch=amd64
[INFO ][20161201 18:31:00,913][Environment:100] Client environment:os.version=6.1
[INFO ][20161201 18:31:00,913][Environment:100] Client environment:user.name=admin
[INFO ][20161201 18:31:00,913][Environment:100] Client environment:user.home=C:\Users\admin
[INFO ][20161201 18:31:00,914][Environment:100] Client environment:user.dir=D:\workspace\fengyu
[INFO ][20161201 18:31:00,915][ZooKeeper:438] Initiating client connection, connectString=139.196.225.215:2181 sessionTimeout=30000 watcher=org.I0Itec.zkclient.ZkClient@3f92c349
[INFO ][20161201 18:31:00,992][ZkClient:934] Waiting for keeper state SyncConnected
[INFO ][20161201 18:31:00,997][ClientCnxn$SendThread:975] Opening socket connection to server 139.196.225.215/139.196.225.215:2181. Will not attempt to authenticate using SASL (unknown error)
[INFO ][20161201 18:31:01,033][ClientCnxn$SendThread:852] Socket connection established to 139.196.225.215/139.196.225.215:2181, initiating session
[INFO ][20161201 18:31:01,067][ClientCnxn$SendThread:1235] Session establishment complete on server 139.196.225.215/139.196.225.215:2181, sessionid = 0x158b9767d180003, negotiated timeout = 30000
[INFO ][20161201 18:31:01,070][ZkClient:711] zookeeper state changed (SyncConnected)
[INFO ][20161201 18:31:01,072][AbstractRegistry:302]  [DUBBO] Register: dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:01,182][AbstractRegistry:325]  [DUBBO] Subscribe: provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:31:01,305][AbstractRegistry:422]  [DUBBO] Notify urls for subscribe url provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262, urls: [empty://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262], dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:43:10,633][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hgeyo19k1fl9wgu1vtc07v|5d8bafa9, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hgeyo19k1fl9wgu1vtc07v|5d8bafa9, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://139.196.225.215:3306/edu_simple_pay?useUnicode=true&characterEncoding=utf-8&allowMultiQueries=true, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 15, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 3, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[WARN ][20161201 18:43:43,137][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@6454c93 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:43,140][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@693d9bc6 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[ERROR][20161201 18:43:43,145][ExceptionFilter:87]  [DUBBO] Got unchecked and undeclared exception which called by 192.168.171.1. service: com.fengyu.facade.user.service.UserFacade, method: findByLoginName, exception: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!, dubbo version: 2.5.3, current host: 127.0.0.1
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:75)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy16.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:163)
	at com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName(UserDaoImpl.java:18)
	at com.fengyu.service.user.biz.UserBiz.findByLoginName(UserBiz.java:22)
	at com.fengyu.facade.user.impl.UserFacadeImpl.findByLoginName(UserFacadeImpl.java:23)
	at com.alibaba.dubbo.common.bytecode.Wrapper1.invokeMethod(Wrapper1.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:46)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:72)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:64)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:42)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:78)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:60)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:112)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:108)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:84)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:170)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:52)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 33 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:67)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:279)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:72)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:59)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 40 more
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:111)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:77)
	... 50 more
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 53 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[ERROR][20161201 18:43:43,179][ExceptionFilter:87]  [DUBBO] Got unchecked and undeclared exception which called by 192.168.171.1. service: com.fengyu.facade.user.service.UserFacade, method: findByLoginName, exception: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!, dubbo version: 2.5.3, current host: 127.0.0.1
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:75)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy16.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:163)
	at com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName(UserDaoImpl.java:18)
	at com.fengyu.service.user.biz.UserBiz.findByLoginName(UserBiz.java:22)
	at com.fengyu.facade.user.impl.UserFacadeImpl.findByLoginName(UserFacadeImpl.java:23)
	at com.alibaba.dubbo.common.bytecode.Wrapper1.invokeMethod(Wrapper1.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:46)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:72)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:64)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:42)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:78)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:60)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:112)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:108)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:84)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:170)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:52)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 33 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:67)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:279)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:72)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:59)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 40 more
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:111)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:77)
	... 50 more
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 53 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[ERROR][20161201 18:43:43,183][ExceptionFilter:87]  [DUBBO] Got unchecked and undeclared exception which called by 192.168.171.1. service: com.fengyu.facade.user.service.UserFacade, method: findByLoginName, exception: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!, dubbo version: 2.5.3, current host: 127.0.0.1
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:75)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy16.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:163)
	at com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName(UserDaoImpl.java:18)
	at com.fengyu.service.user.biz.UserBiz.findByLoginName(UserBiz.java:22)
	at com.fengyu.facade.user.impl.UserFacadeImpl.findByLoginName(UserFacadeImpl.java:23)
	at com.alibaba.dubbo.common.bytecode.Wrapper1.invokeMethod(Wrapper1.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:46)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:72)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:64)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:42)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:78)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:60)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:112)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:108)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:84)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:170)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:52)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 33 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:67)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:279)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:72)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:59)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 40 more
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:111)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:77)
	... 50 more
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 53 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:43,314][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@14d19f62 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:43,315][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@693d9bc6 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:43:43,335][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@7848aebe -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:43,336][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@693d9bc6 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:43:44,861][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@3930041c -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:44,862][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@693d9bc6 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:43:44,928][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@62390ce4 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:44,929][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@693d9bc6 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:43:44,937][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@1c1d25ac -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'edu_simple_pay'
	at sun.reflect.GeneratedConstructorAccessor14.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor12.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:43:44,938][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@693d9bc6 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[INFO ][20161201 18:44:37,394][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1f89ab83: startup date [Thu Dec 01 18:44:37 CST 2016]; root of context hierarchy
[INFO ][20161201 18:44:37,458][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 18:44:37,887][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 18:44:37,919][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 18:44:37,970][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 18:44:38,258][PropertiesLoaderSupport:172] Loading properties file from class path resource [db.properties]
[INFO ][20161201 18:44:38,259][PropertiesLoaderSupport:172] Loading properties file from class path resource [public_system.properties]
[INFO ][20161201 18:44:38,577][Slf4jMLog$Slf4jMLogger$InfoLogger:212] MLog clients using slf4j logging.
[INFO ][20161201 18:44:39,622][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
[INFO ][20161201 18:44:40,252][ServiceBean:107]  [DUBBO] The service ready on spring started. service: com.fengyu.facade.user.service.UserFacade, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:40,376][ServiceConfig:510]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to local registry, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:40,376][ServiceConfig:470]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:40,376][ServiceConfig:481]  [DUBBO] Register dubbo service com.fengyu.facade.user.service.UserFacade url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266 to registry registry://139.196.225.215:2181/com.alibaba.dubbo.registry.RegistryService?application=fengyu-service-user&dubbo=2.5.3&pid=16824&registry=zookeeper&timestamp=1480589080260, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:40,841][AbstractServer:69]  [DUBBO] Start NettyServer bind /0.0.0.0:20881, export /192.168.171.1:20881, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:40,858][AbstractRegistry:232]  [DUBBO] Load registry store file C:\Users\admin\.dubbo\dubbo-registry-139.196.225.215.cache, data: {com.fengyu.facade.user.service.UserFacade=empty://192.168.171.1/com.fengyu.facade.user.service.UserFacade?application=fengyu-web-shop&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15500&revision=1.0.0&side=consumer&timestamp=1480588791908 empty://192.168.171.1/com.fengyu.facade.user.service.UserFacade?application=fengyu-web-shop&category=routers&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15500&revision=1.0.0&side=consumer&timestamp=1480588791908 dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=17032&side=provider&timestamp=1480588260262}, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:40,870][ZkEventThread:64] Starting ZkClient event thread.
[INFO ][20161201 18:44:40,876][Environment:100] Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
[INFO ][20161201 18:44:40,877][Environment:100] Client environment:host.name=windows10.microdone.cn
[INFO ][20161201 18:44:40,877][Environment:100] Client environment:java.version=1.8.0_101
[INFO ][20161201 18:44:40,877][Environment:100] Client environment:java.vendor=Oracle Corporation
[INFO ][20161201 18:44:40,877][Environment:100] Client environment:java.home=D:\Program Files\Java\jdk1.8.0_101\jre
[INFO ][20161201 18:44:40,878][Environment:100] Client environment:java.class.path=D:\Program Files\Java\jdk1.8.0_101\jre\lib\charsets.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\deploy.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\access-bridge-64.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\cldrdata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\dnsns.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jaccess.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jfxrt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\localedata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\nashorn.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunec.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunjce_provider.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunmscapi.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunpkcs11.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\zipfs.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\javaws.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jce.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfr.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfxswt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jsse.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\management-agent.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\plugin.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\resources.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\rt.jar;D:\workspace\fengyu\fengyu-service-user\target\test-classes;D:\workspace\fengyu\fengyu-service-user\target\classes;D:\workspace\fengyu\fengyu-common-core\target\classes;D:\workspace\fengyu\fengyu-common\target\classes;D:\mavenLocal\com\jcraft\jsch\0.1.50\jsch-0.1.50.jar;D:\workspace\fengyu\fengyu-common-config\target\classes;D:\workspace\fengyu\fengyu-facade-user\target\classes;D:\mavenLocal\junit\junit\4.12\junit-4.12.jar;D:\mavenLocal\org\hamcrest\hamcrest-core\1.3\hamcrest-core-1.3.jar;D:\mavenLocal\log4j\log4j\1.2.17\log4j-1.2.17.jar;D:\mavenLocal\net\sf\json-lib\json-lib\2.0\json-lib-2.0-jdk15.jar;D:\mavenLocal\commons-beanutils\commons-beanutils\1.7.0\commons-beanutils-1.7.0.jar;D:\mavenLocal\commons-collections\commons-collections\3.2\commons-collections-3.2.jar;D:\mavenLocal\commons-logging\commons-logging\1.1\commons-logging-1.1.jar;D:\mavenLocal\net\sf\ezmorph\ezmorph\1.0.2\ezmorph-1.0.2.jar;D:\mavenLocal\org\json\json\20151123\json-20151123.jar;D:\mavenLocal\commons-lang\commons-lang\2.6\commons-lang-2.6.jar;D:\mavenLocal\org\springframework\spring-context\4.2.4.RELEASE\spring-context-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aop\4.2.4.RELEASE\spring-aop-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-beans\4.2.4.RELEASE\spring-beans-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-core\4.2.4.RELEASE\spring-core-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-expression\4.2.4.RELEASE\spring-expression-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-context-support\4.2.4.RELEASE\spring-context-support-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aspects\4.2.4.RELEASE\spring-aspects-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-tx\4.2.4.RELEASE\spring-tx-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-jdbc\4.2.4.RELEASE\spring-jdbc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-orm\4.2.4.RELEASE\spring-orm-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-web\4.2.4.RELEASE\spring-web-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-webmvc\4.2.4.RELEASE\spring-webmvc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-test\4.2.4.RELEASE\spring-test-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-remoting\2.0.8\spring-remoting-2.0.8.jar;D:\mavenLocal\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;D:\mavenLocal\org\aspectj\aspectjweaver\1.7.3\aspectjweaver-1.7.3.jar;D:\mavenLocal\cglib\cglib\2.2.2\cglib-2.2.2.jar;D:\mavenLocal\asm\asm\3.3.1\asm-3.3.1.jar;D:\mavenLocal\org\freemarker\freemarker\2.3.23\freemarker-2.3.23.jar;D:\mavenLocal\com\alibaba\druid\1.0.14\druid-1.0.14.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\jconsole.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\tools.jar;D:\mavenLocal\com\mchange\c3p0\0.9.5.2\c3p0-0.9.5.2.jar;D:\mavenLocal\com\mchange\mchange-commons-java\0.2.11\mchange-commons-java-0.2.11.jar;D:\mavenLocal\mysql\mysql-connector-java\5.1.38\mysql-connector-java-5.1.38.jar;D:\mavenLocal\redis\clients\jedis\2.4.2\jedis-2.4.2.jar;D:\mavenLocal\org\apache\commons\commons-pool2\2.0\commons-pool2-2.0.jar;D:\mavenLocal\net\sf\ehcache\ehcache\2.10.1\ehcache-2.10.1.jar;D:\mavenLocal\org\mybatis\mybatis\3.2.8\mybatis-3.2.8.jar;D:\mavenLocal\org\mybatis\mybatis-spring\1.2.2\mybatis-spring-1.2.2.jar;D:\mavenLocal\javax\persistence\persistence-api\1.0\persistence-api-1.0.jar;D:\mavenLocal\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar;D:\mavenLocal\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar;D:\mavenLocal\commons-codec\commons-codec\1.6\commons-codec-1.6.jar;D:\mavenLocal\commons-net\commons-net\3.2\commons-net-3.2.jar;D:\mavenLocal\org\jdom\jdom\1.1.3\jdom-1.1.3.jar;D:\mavenLocal\com\github\pagehelper\pagehelper\4.1.5\pagehelper-4.1.5.jar;D:\mavenLocal\com\github\jsqlparser\jsqlparser\0.9.5\jsqlparser-0.9.5.jar;D:\mavenLocal\javax\servlet\javax.servlet-api\3.1.0\javax.servlet-api-3.1.0.jar;D:\mavenLocal\commons-fileupload\commons-fileupload\1.2.1\commons-fileupload-1.2.1.jar;D:\mavenLocal\commons-io\commons-io\1.3.2\commons-io-1.3.2.jar;D:\mavenLocal\com\alibaba\dubbo\2.5.3\dubbo-2.5.3.jar;D:\mavenLocal\org\javassist\javassist\3.15.0-GA\javassist-3.15.0-GA.jar;D:\mavenLocal\org\jboss\netty\netty\3.2.5.Final\netty-3.2.5.Final.jar;D:\mavenLocal\org\apache\zookeeper\zookeeper\3.4.6\zookeeper-3.4.6.jar;D:\mavenLocal\jline\jline\0.9.94\jline-0.9.94.jar;D:\mavenLocal\io\netty\netty\3.7.0.Final\netty-3.7.0.Final.jar;D:\mavenLocal\com\101tec\zkclient\0.7\zkclient-0.7.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-core\2.4.3\jackson-core-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-databind\2.4.3\jackson-databind-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-annotations\2.4.0\jackson-annotations-2.4.0.jar;D:\mavenLocal\net\sourceforge\jexcelapi\jxl\2.6.12\jxl-2.6.12.jar;D:\mavenLocal\org\jsoup\jsoup\1.7.3\jsoup-1.7.3.jar;D:\mavenLocal\org\apache\tomcat\servlet-api\6.0.37\servlet-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\jsp-api\6.0.37\jsp-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\el-api\6.0.37\el-api-6.0.37.jar;D:\mavenLocal\org\slf4j\slf4j-api\1.7.5\slf4j-api-1.7.5.jar;D:\mavenLocal\org\slf4j\slf4j-log4j12\1.7.5\slf4j-log4j12-1.7.5.jar;D:\Program Files\JetBrains\IntelliJ IDEA 2016.1\lib\idea_rt.jar
[INFO ][20161201 18:44:40,879][Environment:100] Client environment:java.library.path=D:\Program Files\Java\jdk1.8.0_101\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\NetSarang;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x64;C:\Program Files\Lenovo\Fingerprint Manager Pro\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;D:\Program Files\TortoiseSVN\bin;D:\Program Files\Java\jdk1.8.0_101\bin;D:\Program Files\Java\jdk1.8.0_101\jre\bin;D:\maven-3.0.5/bin;D:\maven-3.0.5\bin;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;.
[INFO ][20161201 18:44:40,879][Environment:100] Client environment:java.io.tmpdir=C:\Users\admin\AppData\Local\Temp\
[INFO ][20161201 18:44:40,879][Environment:100] Client environment:java.compiler=<NA>
[INFO ][20161201 18:44:40,880][Environment:100] Client environment:os.name=Windows 7
[INFO ][20161201 18:44:40,880][Environment:100] Client environment:os.arch=amd64
[INFO ][20161201 18:44:40,880][Environment:100] Client environment:os.version=6.1
[INFO ][20161201 18:44:40,880][Environment:100] Client environment:user.name=admin
[INFO ][20161201 18:44:40,880][Environment:100] Client environment:user.home=C:\Users\admin
[INFO ][20161201 18:44:40,880][Environment:100] Client environment:user.dir=D:\workspace\fengyu
[INFO ][20161201 18:44:40,881][ZooKeeper:438] Initiating client connection, connectString=139.196.225.215:2181 sessionTimeout=30000 watcher=org.I0Itec.zkclient.ZkClient@576f63f6
[INFO ][20161201 18:44:40,969][ZkClient:934] Waiting for keeper state SyncConnected
[INFO ][20161201 18:44:40,973][ClientCnxn$SendThread:975] Opening socket connection to server 139.196.225.215/139.196.225.215:2181. Will not attempt to authenticate using SASL (unknown error)
[INFO ][20161201 18:44:41,009][ClientCnxn$SendThread:852] Socket connection established to 139.196.225.215/139.196.225.215:2181, initiating session
[INFO ][20161201 18:44:41,063][ClientCnxn$SendThread:1235] Session establishment complete on server 139.196.225.215/139.196.225.215:2181, sessionid = 0x158b9767d180005, negotiated timeout = 30000
[INFO ][20161201 18:44:41,065][ZkClient:711] zookeeper state changed (SyncConnected)
[INFO ][20161201 18:44:41,066][AbstractRegistry:302]  [DUBBO] Register: dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:41,193][AbstractRegistry:325]  [DUBBO] Subscribe: provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:41,347][AbstractRegistry:422]  [DUBBO] Notify urls for subscribe url provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266, urls: [empty://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266], dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:44:52,204][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hgeyo19k1flrh68zltb9h|3b0090a4, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hgeyo19k1flrh68zltb9h|3b0090a4, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://139.196.225.215:3306/fengyu-dubbo?useUnicode=true&characterEncoding=utf-8&allowMultiQueries=true, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 15, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 3, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[WARN ][20161201 18:45:24,707][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@4a9c08d5 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:24,709][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@7347916d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[ERROR][20161201 18:45:24,713][ExceptionFilter:87]  [DUBBO] Got unchecked and undeclared exception which called by 192.168.171.1. service: com.fengyu.facade.user.service.UserFacade, method: findByLoginName, exception: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!, dubbo version: 2.5.3, current host: 127.0.0.1
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:75)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy16.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:163)
	at com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName(UserDaoImpl.java:18)
	at com.fengyu.service.user.biz.UserBiz.findByLoginName(UserBiz.java:22)
	at com.fengyu.facade.user.impl.UserFacadeImpl.findByLoginName(UserFacadeImpl.java:24)
	at com.alibaba.dubbo.common.bytecode.Wrapper1.invokeMethod(Wrapper1.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:46)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:72)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:64)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:42)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:78)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:60)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:112)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:108)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:84)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:170)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:52)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 33 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:67)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:279)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:72)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:59)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 40 more
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:111)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:77)
	... 50 more
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 53 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[ERROR][20161201 18:45:24,713][ExceptionFilter:87]  [DUBBO] Got unchecked and undeclared exception which called by 192.168.171.1. service: com.fengyu.facade.user.service.UserFacade, method: findByLoginName, exception: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!, dubbo version: 2.5.3, current host: 127.0.0.1
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:75)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy16.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:163)
	at com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName(UserDaoImpl.java:18)
	at com.fengyu.service.user.biz.UserBiz.findByLoginName(UserBiz.java:22)
	at com.fengyu.facade.user.impl.UserFacadeImpl.findByLoginName(UserFacadeImpl.java:24)
	at com.alibaba.dubbo.common.bytecode.Wrapper1.invokeMethod(Wrapper1.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:46)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:72)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:64)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:42)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:78)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:60)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:112)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:108)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:84)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:170)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:52)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 33 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:67)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:279)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:72)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:59)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 40 more
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:111)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:77)
	... 50 more
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 53 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[ERROR][20161201 18:45:24,713][ExceptionFilter:87]  [DUBBO] Got unchecked and undeclared exception which called by 192.168.171.1. service: com.fengyu.facade.user.service.UserFacade, method: findByLoginName, exception: org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!, dubbo version: 2.5.3, current host: 127.0.0.1
org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:75)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	at com.sun.proxy.$Proxy16.selectOne(Unknown Source)
	at org.mybatis.spring.SqlSessionTemplate.selectOne(SqlSessionTemplate.java:163)
	at com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName(UserDaoImpl.java:18)
	at com.fengyu.service.user.biz.UserBiz.findByLoginName(UserBiz.java:22)
	at com.fengyu.facade.user.impl.UserFacadeImpl.findByLoginName(UserFacadeImpl.java:24)
	at com.alibaba.dubbo.common.bytecode.Wrapper1.invokeMethod(Wrapper1.java)
	at com.alibaba.dubbo.rpc.proxy.javassist.JavassistProxyFactory$1.doInvoke(JavassistProxyFactory.java:46)
	at com.alibaba.dubbo.rpc.proxy.AbstractProxyInvoker.invoke(AbstractProxyInvoker.java:72)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.filter.ExceptionFilter.invoke(ExceptionFilter.java:64)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.TimeoutFilter.invoke(TimeoutFilter.java:42)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.TraceFilter.invoke(TraceFilter.java:78)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ContextFilter.invoke(ContextFilter.java:60)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.GenericFilter.invoke(GenericFilter.java:112)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ClassLoaderFilter.invoke(ClassLoaderFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.EchoFilter.invoke(EchoFilter.java:38)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboProtocol$1.reply(DubboProtocol.java:108)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.handleRequest(HeaderExchangeHandler.java:84)
	at com.alibaba.dubbo.remoting.exchange.support.header.HeaderExchangeHandler.received(HeaderExchangeHandler.java:170)
	at com.alibaba.dubbo.remoting.transport.DecodeHandler.received(DecodeHandler.java:52)
	at com.alibaba.dubbo.remoting.transport.dispatcher.ChannelEventRunnable.run(ChannelEventRunnable.java:82)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### The error may exist in mybatis/mappings/userMapper.xml
### The error may involve com.fengyu.service.user.dao.impl.UserDaoImpl.findByLoginName
### The error occurred while executing a query
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:26)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:111)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectOne(DefaultSqlSession.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	... 33 more
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:80)
	at org.mybatis.spring.transaction.SpringManagedTransaction.openConnection(SpringManagedTransaction.java:81)
	at org.mybatis.spring.transaction.SpringManagedTransaction.getConnection(SpringManagedTransaction.java:67)
	at org.apache.ibatis.executor.BaseExecutor.getConnection(BaseExecutor.java:279)
	at org.apache.ibatis.executor.SimpleExecutor.prepareStatement(SimpleExecutor.java:72)
	at org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:59)
	at org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	at org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:96)
	at org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	... 40 more
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.springframework.jdbc.datasource.DataSourceUtils.doGetConnection(DataSourceUtils.java:111)
	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:77)
	... 50 more
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 53 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:24,728][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@4e99f7bc -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:24,764][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@7347916d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:45:24,765][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@761b8eb9 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:24,767][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@7347916d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:45:26,353][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@68b42b0b -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:26,353][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@7347916d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:45:26,379][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@23bf03a4 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:26,380][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@7347916d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[WARN ][20161201 18:45:26,437][Slf4jMLog$Slf4jMLogger$WarnLogger:223] com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@744d6280 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'fengyu-dubbo'
	at sun.reflect.GeneratedConstructorAccessor16.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.Util.getInstance(Util.java:387)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:939)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3878)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3814)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:871)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:4323)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1267)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2255)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2286)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2085)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:795)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor11.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:404)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:400)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:327)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
[WARN ][20161201 18:45:26,438][Slf4jMLog$Slf4jMLogger$WarnLogger:220] Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@7347916d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
[INFO ][20161201 18:46:34,589][AbstractApplicationContext:578] Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@1dfe2924: startup date [Thu Dec 01 18:46:34 CST 2016]; root of context hierarchy
[INFO ][20161201 18:46:34,691][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-context.xml]
[INFO ][20161201 18:46:35,304][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/spring-mybatis.xml]
[INFO ][20161201 18:46:35,386][XmlBeanDefinitionReader:317] Loading XML bean definitions from class path resource [spring/dubbo-provider.xml]
[INFO ][20161201 18:46:35,428][?:?] using logger: com.alibaba.dubbo.common.logger.log4j.Log4jLoggerAdapter
[INFO ][20161201 18:46:36,073][PropertiesLoaderSupport:172] Loading properties file from class path resource [db.properties]
[INFO ][20161201 18:46:36,074][PropertiesLoaderSupport:172] Loading properties file from class path resource [public_system.properties]
[INFO ][20161201 18:46:37,048][Slf4jMLog$Slf4jMLogger$InfoLogger:212] MLog clients using slf4j logging.
[INFO ][20161201 18:46:37,492][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
[INFO ][20161201 18:46:38,166][ServiceBean:107]  [DUBBO] The service ready on spring started. service: com.fengyu.facade.user.service.UserFacade, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:38,308][ServiceConfig:510]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to local registry, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:38,308][ServiceConfig:470]  [DUBBO] Export dubbo service com.fengyu.facade.user.service.UserFacade to url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:38,309][ServiceConfig:481]  [DUBBO] Register dubbo service com.fengyu.facade.user.service.UserFacade url dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181 to registry registry://139.196.225.215:2181/com.alibaba.dubbo.registry.RegistryService?application=fengyu-service-user&dubbo=2.5.3&pid=15564&registry=zookeeper&timestamp=1480589198173, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:38,875][AbstractServer:69]  [DUBBO] Start NettyServer bind /0.0.0.0:20881, export /192.168.171.1:20881, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:38,917][AbstractRegistry:232]  [DUBBO] Load registry store file C:\Users\admin\.dubbo\dubbo-registry-139.196.225.215.cache, data: {com.fengyu.facade.user.service.UserFacade=empty://192.168.171.1/com.fengyu.facade.user.service.UserFacade?application=fengyu-web-shop&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15500&revision=1.0.0&side=consumer&timestamp=1480588791908 empty://192.168.171.1/com.fengyu.facade.user.service.UserFacade?application=fengyu-web-shop&category=routers&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15500&revision=1.0.0&side=consumer&timestamp=1480588791908 dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=16824&side=provider&timestamp=1480589080266}, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:38,959][ZkEventThread:64] Starting ZkClient event thread.
[INFO ][20161201 18:46:38,968][Environment:100] Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
[INFO ][20161201 18:46:38,971][Environment:100] Client environment:host.name=windows10.microdone.cn
[INFO ][20161201 18:46:38,971][Environment:100] Client environment:java.version=1.8.0_101
[INFO ][20161201 18:46:38,971][Environment:100] Client environment:java.vendor=Oracle Corporation
[INFO ][20161201 18:46:38,972][Environment:100] Client environment:java.home=D:\Program Files\Java\jdk1.8.0_101\jre
[INFO ][20161201 18:46:38,972][Environment:100] Client environment:java.class.path=D:\Program Files\Java\jdk1.8.0_101\jre\lib\charsets.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\deploy.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\access-bridge-64.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\cldrdata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\dnsns.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jaccess.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\jfxrt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\localedata.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\nashorn.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunec.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunjce_provider.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunmscapi.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\sunpkcs11.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\ext\zipfs.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\javaws.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jce.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfr.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jfxswt.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\jsse.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\management-agent.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\plugin.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\resources.jar;D:\Program Files\Java\jdk1.8.0_101\jre\lib\rt.jar;D:\workspace\fengyu\fengyu-service-user\target\test-classes;D:\workspace\fengyu\fengyu-service-user\target\classes;D:\workspace\fengyu\fengyu-common-core\target\classes;D:\workspace\fengyu\fengyu-common\target\classes;D:\mavenLocal\com\jcraft\jsch\0.1.50\jsch-0.1.50.jar;D:\workspace\fengyu\fengyu-common-config\target\classes;D:\workspace\fengyu\fengyu-facade-user\target\classes;D:\mavenLocal\junit\junit\4.12\junit-4.12.jar;D:\mavenLocal\org\hamcrest\hamcrest-core\1.3\hamcrest-core-1.3.jar;D:\mavenLocal\log4j\log4j\1.2.17\log4j-1.2.17.jar;D:\mavenLocal\net\sf\json-lib\json-lib\2.0\json-lib-2.0-jdk15.jar;D:\mavenLocal\commons-beanutils\commons-beanutils\1.7.0\commons-beanutils-1.7.0.jar;D:\mavenLocal\commons-collections\commons-collections\3.2\commons-collections-3.2.jar;D:\mavenLocal\commons-logging\commons-logging\1.1\commons-logging-1.1.jar;D:\mavenLocal\net\sf\ezmorph\ezmorph\1.0.2\ezmorph-1.0.2.jar;D:\mavenLocal\org\json\json\20151123\json-20151123.jar;D:\mavenLocal\commons-lang\commons-lang\2.6\commons-lang-2.6.jar;D:\mavenLocal\org\springframework\spring-context\4.2.4.RELEASE\spring-context-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aop\4.2.4.RELEASE\spring-aop-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-beans\4.2.4.RELEASE\spring-beans-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-core\4.2.4.RELEASE\spring-core-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-expression\4.2.4.RELEASE\spring-expression-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-context-support\4.2.4.RELEASE\spring-context-support-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-aspects\4.2.4.RELEASE\spring-aspects-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-tx\4.2.4.RELEASE\spring-tx-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-jdbc\4.2.4.RELEASE\spring-jdbc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-orm\4.2.4.RELEASE\spring-orm-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-web\4.2.4.RELEASE\spring-web-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-webmvc\4.2.4.RELEASE\spring-webmvc-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-test\4.2.4.RELEASE\spring-test-4.2.4.RELEASE.jar;D:\mavenLocal\org\springframework\spring-remoting\2.0.8\spring-remoting-2.0.8.jar;D:\mavenLocal\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;D:\mavenLocal\org\aspectj\aspectjweaver\1.7.3\aspectjweaver-1.7.3.jar;D:\mavenLocal\cglib\cglib\2.2.2\cglib-2.2.2.jar;D:\mavenLocal\asm\asm\3.3.1\asm-3.3.1.jar;D:\mavenLocal\org\freemarker\freemarker\2.3.23\freemarker-2.3.23.jar;D:\mavenLocal\com\alibaba\druid\1.0.14\druid-1.0.14.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\jconsole.jar;D:\Program Files\Java\jdk1.8.0_101\jre\..\lib\tools.jar;D:\mavenLocal\com\mchange\c3p0\0.9.5.2\c3p0-0.9.5.2.jar;D:\mavenLocal\com\mchange\mchange-commons-java\0.2.11\mchange-commons-java-0.2.11.jar;D:\mavenLocal\mysql\mysql-connector-java\5.1.38\mysql-connector-java-5.1.38.jar;D:\mavenLocal\redis\clients\jedis\2.4.2\jedis-2.4.2.jar;D:\mavenLocal\org\apache\commons\commons-pool2\2.0\commons-pool2-2.0.jar;D:\mavenLocal\net\sf\ehcache\ehcache\2.10.1\ehcache-2.10.1.jar;D:\mavenLocal\org\mybatis\mybatis\3.2.8\mybatis-3.2.8.jar;D:\mavenLocal\org\mybatis\mybatis-spring\1.2.2\mybatis-spring-1.2.2.jar;D:\mavenLocal\javax\persistence\persistence-api\1.0\persistence-api-1.0.jar;D:\mavenLocal\org\apache\httpcomponents\httpclient\4.3.3\httpclient-4.3.3.jar;D:\mavenLocal\org\apache\httpcomponents\httpcore\4.3.2\httpcore-4.3.2.jar;D:\mavenLocal\commons-codec\commons-codec\1.6\commons-codec-1.6.jar;D:\mavenLocal\commons-net\commons-net\3.2\commons-net-3.2.jar;D:\mavenLocal\org\jdom\jdom\1.1.3\jdom-1.1.3.jar;D:\mavenLocal\com\github\pagehelper\pagehelper\4.1.5\pagehelper-4.1.5.jar;D:\mavenLocal\com\github\jsqlparser\jsqlparser\0.9.5\jsqlparser-0.9.5.jar;D:\mavenLocal\javax\servlet\javax.servlet-api\3.1.0\javax.servlet-api-3.1.0.jar;D:\mavenLocal\commons-fileupload\commons-fileupload\1.2.1\commons-fileupload-1.2.1.jar;D:\mavenLocal\commons-io\commons-io\1.3.2\commons-io-1.3.2.jar;D:\mavenLocal\com\alibaba\dubbo\2.5.3\dubbo-2.5.3.jar;D:\mavenLocal\org\javassist\javassist\3.15.0-GA\javassist-3.15.0-GA.jar;D:\mavenLocal\org\jboss\netty\netty\3.2.5.Final\netty-3.2.5.Final.jar;D:\mavenLocal\org\apache\zookeeper\zookeeper\3.4.6\zookeeper-3.4.6.jar;D:\mavenLocal\jline\jline\0.9.94\jline-0.9.94.jar;D:\mavenLocal\io\netty\netty\3.7.0.Final\netty-3.7.0.Final.jar;D:\mavenLocal\com\101tec\zkclient\0.7\zkclient-0.7.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-core\2.4.3\jackson-core-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-databind\2.4.3\jackson-databind-2.4.3.jar;D:\mavenLocal\com\fasterxml\jackson\core\jackson-annotations\2.4.0\jackson-annotations-2.4.0.jar;D:\mavenLocal\net\sourceforge\jexcelapi\jxl\2.6.12\jxl-2.6.12.jar;D:\mavenLocal\org\jsoup\jsoup\1.7.3\jsoup-1.7.3.jar;D:\mavenLocal\org\apache\tomcat\servlet-api\6.0.37\servlet-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\jsp-api\6.0.37\jsp-api-6.0.37.jar;D:\mavenLocal\org\apache\tomcat\el-api\6.0.37\el-api-6.0.37.jar;D:\mavenLocal\org\slf4j\slf4j-api\1.7.5\slf4j-api-1.7.5.jar;D:\mavenLocal\org\slf4j\slf4j-log4j12\1.7.5\slf4j-log4j12-1.7.5.jar;D:\Program Files\JetBrains\IntelliJ IDEA 2016.1\lib\idea_rt.jar
[INFO ][20161201 18:46:38,973][Environment:100] Client environment:java.library.path=D:\Program Files\Java\jdk1.8.0_101\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\NetSarang;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x86;C:\Program Files (x86)\Intel\OpenCL SDK\3.0\bin\x64;C:\Program Files\Lenovo\Fingerprint Manager Pro\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;D:\Program Files\TortoiseSVN\bin;D:\Program Files\Java\jdk1.8.0_101\bin;D:\Program Files\Java\jdk1.8.0_101\jre\bin;D:\maven-3.0.5/bin;D:\maven-3.0.5\bin;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;.
[INFO ][20161201 18:46:38,973][Environment:100] Client environment:java.io.tmpdir=C:\Users\admin\AppData\Local\Temp\
[INFO ][20161201 18:46:38,973][Environment:100] Client environment:java.compiler=<NA>
[INFO ][20161201 18:46:38,974][Environment:100] Client environment:os.name=Windows 7
[INFO ][20161201 18:46:38,974][Environment:100] Client environment:os.arch=amd64
[INFO ][20161201 18:46:38,974][Environment:100] Client environment:os.version=6.1
[INFO ][20161201 18:46:38,974][Environment:100] Client environment:user.name=admin
[INFO ][20161201 18:46:38,974][Environment:100] Client environment:user.home=C:\Users\admin
[INFO ][20161201 18:46:38,974][Environment:100] Client environment:user.dir=D:\workspace\fengyu
[INFO ][20161201 18:46:38,975][ZooKeeper:438] Initiating client connection, connectString=139.196.225.215:2181 sessionTimeout=30000 watcher=org.I0Itec.zkclient.ZkClient@55f8669d
[INFO ][20161201 18:46:39,133][ZkClient:934] Waiting for keeper state SyncConnected
[INFO ][20161201 18:46:39,141][ClientCnxn$SendThread:975] Opening socket connection to server 139.196.225.215/139.196.225.215:2181. Will not attempt to authenticate using SASL (unknown error)
[INFO ][20161201 18:46:39,169][ClientCnxn$SendThread:852] Socket connection established to 139.196.225.215/139.196.225.215:2181, initiating session
[INFO ][20161201 18:46:39,205][ClientCnxn$SendThread:1235] Session establishment complete on server 139.196.225.215/139.196.225.215:2181, sessionid = 0x158b9767d180006, negotiated timeout = 30000
[INFO ][20161201 18:46:39,207][ZkClient:711] zookeeper state changed (SyncConnected)
[INFO ][20161201 18:46:39,209][AbstractRegistry:302]  [DUBBO] Register: dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:39,327][AbstractRegistry:325]  [DUBBO] Subscribe: provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181, dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:39,456][AbstractRegistry:422]  [DUBBO] Notify urls for subscribe url provider://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181, urls: [empty://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&category=configurators&check=false&dubbo=2.5.3&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181], dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:46:52,866][Slf4jMLog$Slf4jMLogger$InfoLogger:212] Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hgeyo19k1flu0ixlgmz3l|5d9b7a8a, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hgeyo19k1flu0ixlgmz3l|5d9b7a8a, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://localhost:3306/fengyu-dubbo?useUnicode=true&characterEncoding=utf-8&allowMultiQueries=true, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 15, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 3, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
[WARN ][20161201 18:47:28,266][AbstractServer:197]  [DUBBO] All clients has discontected from /192.168.171.1:20881. You can graceful shutdown now., dubbo version: 2.5.3, current host: 127.0.0.1
[INFO ][20161201 18:47:28,268][DubboProtocol$1:130]  [DUBBO] disconected from /192.168.171.1:50700,url:dubbo://192.168.171.1:20881/com.fengyu.facade.user.service.UserFacade?anyhost=true&application=fengyu-service-user&channel.readonly.sent=true&codec=dubbo&dubbo=2.5.3&heartbeat=60000&interface=com.fengyu.facade.user.service.UserFacade&methods=findByLoginName&pid=15564&side=provider&timestamp=1480589198181, dubbo version: 2.5.3, current host: 127.0.0.1
